### ISSUE TITLE
No unique fragment ends found for capture length

### ISSUE THREAD

**Issue by KSchtz** [deLaatLab/pipe4C#21]
Hi,

my issue is similar to the one in #20, but recreating the fragment file does not resolve the error. 

_### ERROR: set_1_viewpoint_10_ESC_replicate_1 No unique fragment ends found for capture length._

I'm using the data from the example folder and the provided global config file in which I only changed the paths.

I tried with index data downloaded from https://benlangmead.github.io/aws-indexes/bowtie and with index files generated with my local bowtie.

I'm using
bowtie2 v2.3.0
samtools v1.3.1
R v3.5.1 (session info attached)

I'm new to 4C, so maybe this is a configuration issue, but I tried different settings in the conf.yml before switching back to default.

Any help would be appreciated. Thanks in advance!

[out.txt](https://github.com/deLaatLab/pipe4C/files/8859832/out.txt)
[Log_2022_06_08__09_41_25.txt](https://github.com/deLaatLab/pipe4C/files/8859833/Log_2022_06_08__09_41_25.txt)
[Bowtie2Log_2022_06_08__09_41_25.txt](https://github.com/deLaatLab/pipe4C/files/8859836/Bowtie2Log_2022_06_08__09_41_25.txt)
[R_SessionInfo.txt](https://github.com/deLaatLab/pipe4C/files/8859837/R_SessionInfo.txt)

**Comment by krijgerp** [comment]
Hi KSchtz,
First remove the old fragment genome. In your log file it does not say Creating new digest for genome", meaning you are loading a previously (probally corrupt) file. It should be stored in the fragFolder path indicated in the conf.yml.

If this does not work let me know.
Good luck,
Peter

**Comment by KSchtz** [comment]
Hi Peter,
while I created the fragment file multiple times before without anything changing I only now noticed that the R packages were off last time I created. 

The old log shows a warning:
_Warning message:
no function found corresponding to methods exports from ‘XVector’ for: ‘concatenateObjects’_ 

_BiocManager::valid()_ listed the outdated packages, updating fixed that and recreating the fragment file now resolved the issue. So it was the same issue as in #20. Sorry for bothering you.

Thanks and best regards,
Kristin
